templates/experiments/chat/chat_ui.html

```diff
@@ -32,7 +32,7 @@
 </div>
 {% include 'experiments/chat/input_bar.html' %}
 {% if experiment.consent_form or experiment.post_survey %}
-  <div class="w-full text-center bg-gray-300">
+  <div class="w-full text-center bg-gray-300 dark:bg-gray-800">
```

Comment:
use bg-base classes instead of specific colors where possible
```suggestion
  <div class="w-full text-center bg-base-300">
```